src/common/libzmqutil/cert.c

```diff
@@ -29,7 +29,9 @@
 
 #include "src/common/libutil/errno_safe.h"
 #include "src/common/libutil/strstrip.h"
-#include "src/common/libutil/strlcpy.h"
+#ifndef HAVE_STRLCPY
+#include "src/common/libmissing/strlcpy.h"
+#endif
```

Comment:
This doesn't follow the instructions in the added `src/common/libmissing/Makefile.am`:
```
# or if there is an existing system include just for the missing function
#   #ifdef HAVE_DOIT
#   #include <doit.h>
#   #else
#   #include "src/common/libmissing/doit.h"
#   #endif
```

Should this instead be:
```c
#ifdef HAVE_STRLCPY
#include <strlcpy.h>
#else
#include "src/common/libmissing/strlcpy.h
#endif
```

Reply:
It's provided on bsd-derived systems in `string.h`, which is included earlier, so OK as is?

Reply:
Ah, I had forgotten that. And your comment does say "an existing include just for the missing function" so this was a thinko on my part. Sorry!